src/sipmpdf/functions.py

```diff
@@ -110,13 +112,38 @@ def normal_cdf(x: np.float64,
 
 
 @expand_shape
-def normal_smeared_poisson(x: np.float64,
-                           pedestal: np.float64,
-                           gain: np.float64,
-                           common_noise: np.float64,
-                           pixel_noise: np.float64,
-                           poisson_mean: np.float64,
-                           poisson_borel: np.float64 = 0) -> np.float64:
+def sipm_response_no_dark_no_ap(x: np.float64,
+                                pedestal: np.float64,
+                                gain: np.float64,
+                                common_noise: np.float64,
+                                pixel_noise: np.float64,
+                                poisson_mean: np.float64,
+                                poisson_borel: np.float64 = 0) -> np.float64:
+  """
+  Testing function to give sipm response with no dark current or afterpulsing 
+
+  Parameters
+  ----------
+  x : numpy.float64
+      The observable value
+  pedestal : np.float64
+      The pedestal value
+  gain : np.float64
+      The gain of the SiPM
+  common_noise : np.float64
+      Common factor noise corresponding to sigma_0
```

Comment:
To not explicitly put paper terms in the documentation string (sigma_0 means nothing to someone who is not following our convention). So remove "corresponding to sigma_"